### ISSUE TITLE
highway formula invalid .pc causes segfault in pkg-config

### ISSUE THREAD

**Issue by ViRb3** [Homebrew/homebrew-core#113686]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
brew config  
                                                                                                                                                                                                                                                                                                   
HOMEBREW_VERSION: 3.6.6-32-g2bec760
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2bec760529343832be00662ed99151ddbf41b486
Last commit: 16 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 0b1ef3e865aaa4af1a960ff6e8c3a47d9a25da29
Core tap last commit: 31 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: /usr/local/bin/editor
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.38.1 => /opt/homebrew/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6-arm64
CLT: 14.0.0.0.1.1661618636
Xcode: 14.0.1
Rosetta 2: false

brew doctor 
                                                                                                                                                                                                                                                                                                    
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libtriton.dylib
```


### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

brew install pkg-config highway
pkg-config --list-all

### What happened (include all command output)?
```
[1]    7927 segmentation fault  pkg-config --list-all
```
### What did you expect to happen?

List is printed

### Step-by-step reproduction instructions (by running `brew` commands)

If you run `pkg-config --list-all --debug` it shows the culprit:

```
Looking for package '/opt/homebrew/lib/pkgconfig/libhwy-test.pc'
Considering '/opt/homebrew/lib/pkgconfig/libhwy-test.pc' to be a filename rather than a package name
Reading '/opt/homebrew/lib/pkgconfig/libhwy-test.pc' from file '/opt/homebrew/lib/pkgconfig/libhwy-test.pc'
Parsing package file '/opt/homebrew/lib/pkgconfig/libhwy-test.pc'
  line>prefix=/opt/homebrew/Cellar/highway/1.0.1
 Variable declaration, 'prefix' has value '/opt/homebrew/Cellar/highway/1.0.1'
  line>libdir=${exec_prefix}/lib
[1]    7953 segmentation fault  pkg-config --list-all --debug
```

Looking at the file, the issue is obvious - an invalid variable is used. Diff for fix below:

```diff
prefix=/opt/homebrew/Cellar/highway/1.0.1
-libdir=${exec_prefix}/lib
+libdir=${prefix}/lib
includedir=${prefix}/include

Name: libhwy-test
Description: Efficient and performance-portable SIMD wrapper, test helpers.
Requires: gtest
Version: 1.0.1
Libs: -L${libdir} -lhwy_test
Cflags: -I${includedir}
```

Sadly I have no clue where this file comes from so I can't submit a PR.

**Comment by carlocab** [comment]
Do
```
brew update && brew reinstall highway
```
to pick up the fix.

**Comment by ViRb3** [comment]
Worked, thanks. Out of curiosity, why did I need to do a force reinstall? Was this fix not released in a version bump?

**Comment by carlocab** [comment]
It was not. This was fixed in #112787. `brew upgrade` would pick it up if we bumped the `revision`.

However, we're trying to avoid `revision` bumps these days since lots of users complain about unexpected upgrades to large numbers of formulae when doing `brew install`, which `revision` bumps can trigger.
